src/utils/Actors.util.ts

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { ActorRuntimeOptions } from "../types/actors/ActorRuntimeOptions";
+
+/**
+ * Get response object for /dapr/config API call.
+ * See https://docs.dapr.io/reference/api/actors_api/#get-registered-actors
+ */
+export function getRegisteredActorResponse(entities: string[], options: ActorRuntimeOptions): {[k: string]: any} {
```

Comment:
This method feels a bit bloated / complex for what it should do I think? Isn't there a way that we can create the response more easily? (e.g. by specifying which keys to copy and just copy them over if they are set?)

Other than that general remarks if we can't simplify it:
* Please no single line ifs
* Let's use our KeyValueType if possible
* Let's maybe create a new type for this specifically?

Reply:
>by specifying which keys to copy and just copy them over if they are set? 


One of the motivation was to not have the two directly coupled, i.e., `options.XYZ` can be mapped to `options.actorXYZ`. Since we are not doing it today, i.e. we have 1:1 mapping, let me update this.

Reply:
We can technically remove the util also now, just that the unit test cases are a definite way to know we didn't mess up the de-structuring and adding.